acymailing.com.acymailer.json

```diff
@@ -0,0 +1,34 @@
+{
+    "providerId": "acymailing.com",
+    "providerName": "AcyMailer",
+    "serviceId": "acymailer",
+    "serviceName": "AcyMailer DKIM Authentication",
+    "version": 1,
+    "logoUrl": "https://www.acymailing.com/wp-content/uploads/2019/05/ACYMAILING_logo_mobile.png",
+    "description": "Allows user to easily set up CNAME record for AcyMailer service",
+    "syncPubKeyDomain": "acymailer.com",
+    "syncRedirectDomain": "api.acymailer.com, acymailer.com",
+    "variableDescription": "mappingValue1, mappingValue2 and mappingValue3 are the values assigned to host websiteValue1, websiteValue2 and websiteValue3",
+    "sharedProviderName": true,
+    "syncBlock": true,
```

Comment:
Sure about it? It will imply only async protocol which is not supported by every DNS provider and more problematic onboarding. Also there are no groups, so obviously no multi-stage configuration which requires async process.

Reply:
Indeed I wanted a synchronous protocol but in the common feedback and the documentation it's written that if we don't specify it, then the `synchronous protocol must be disabled for this template`.  
I've removed it.